src/app/core/services/profile.service.ts

```diff
@@ -20,6 +20,15 @@ export class ProfileService {
       .then(res => res.json())
       .then(json => json.profiles || [])
       .then(profiles => {
+
+        // Will remove once profiles.json is updated
+        profiles.map(profile => {
+          if (profile['encodedText']) {
+            profile['repoName'] = profile['encodedText'];
+            delete profile['encodedText'];
+          }
+        });
+
```

Comment:
Will remove this once new name has been accepted and profiles.json is updated. Require this to pass test cases for now

Reply:
@zatkiller the new name is accepted, I would just need to change the profiles as needed. 

Once this PR is merged, I will make the according changes :+1:

Reply:
Actually should we change profiles.json first? I realized if we remove this block before profiles is changed, the CI will still fail

Reply:
Actually should we change the profiles.json first? I realized the CI will fail during the git push if we remove this block first